settings.php

```diff
@@ -86,24 +87,23 @@
  * of the serialized database credentials will be used as a fallback salt.
  *
  * For enhanced security, you may set this variable to a value using the
- * contents of a file outside your docroot that is never saved together
- * with any backups of your Backdrop files and database.
+ * contents of a file outside your docroot that is never saved together with any
+ * backups of your Backdrop files and database.
```

Comment:
*The contents of a file* would be a list of paragraphs/sections in a text file. What a file contains, considered globally is *the file content* or *the content of a file*.  
(I know *contents* is actually used from the setting.php file, but since this part is changed…)